pages/apim/3.x/user-guide/publisher/design-studio/design-studio-create.adoc

```diff
@@ -55,6 +55,15 @@ In the *CONFIGURATION* tab, select *Best match* if you want APIM to match your f
 
 image:{% link images/apim/3.x/api-publisher-guide/design-studio/configuration-tab.png %}[]
 
+Best match flow is chosen if the request matches the flow. A text plain part of the path will take precedence over a path parameter.
```

Comment:
I wonder if we should add a sentence to explain that the deepest path is used.
For example, `/test/subtest/:id` takes precedence over '/test/subtest'